core/src/main/scala/org/apache/spark/storage/memory/MemoryStore.scala

```diff
@@ -385,9 +390,33 @@ private[spark] class MemoryStore(
     }
   }
 
+  def manual_close[T <: MemoryEntry[_]](entry: T): T = {
```

Comment:
Nit: `manualClose`

Reply:
@srowen , fixed in latest commit, sorry for that